localstack-core/localstack/services/cloudformation/resource_provider.py

```diff
@@ -559,7 +560,7 @@ def execute_action(
                 raise NotImplementedError(change_type)  # TODO: change error type
 
     @staticmethod
-    def try_load_resource_provider(resource_type: str) -> ResourceProvider | None:
+    def try_load_resource_provider(action: str, resource_type: str) -> ResourceProvider | None:
```

Comment:
nit (but mostly just a comment): I get the intent but it does feel a bit weird passing it in here like this just for analytics purposes.